File: Find First Digit/find_first_digit.c
#include <stdio.h>
#include <stdlib.h>
int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    int temp = abs(n);
    while (temp >= 10) {
        temp = temp / 10;
    }
    printf("The first digit is: %d\n", temp);
    return 0;
}

File: Find First Digit/FindFirstDigit.java
public class FindFirstDigit {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();
        int temp = Math.abs(n);
        while (temp >= 10) {
            temp = temp / 10;
        }
        System.out.println("The first digit is: " + temp);
    }
}
